presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -485,7 +486,7 @@ private void startInBackground(QueryExecution query)
                 group = group.parent.get();
             }
             updateEligiblility();
-            executor.execute(() -> query.start(Optional.of(id.toString())));
+            executor.execute(() -> query.start());
```

Comment:
Ya, I more or less reverted that commit. I think this is a better way to expose resource group id anyways. So this diff is sort of just doing that (revert and reimplement). The second diff used this change to find resource group from query manager.